module_utils/network_lsr/argument_validator.py

```diff
@@ -72,6 +76,7 @@ def connection_get_non_absent_names(connections):
 
 class ValidationError(MyError):
     def __init__(self, name, message):
+        # pylint: disable=non-parent-init-called
         Exception.__init__(self, name + ": " + message)
```

Comment:
Is it possible to use `super()` here instead of ignoring the warning? https://docs.python.org/2.7/library/functions.html#super

Reply:
Part of the problem is that the superclass is `MyError` but the code is calling `Exception.__init__` - I have never seen a constructor bypass the superclass and use an ancestor class instead - I figured this was odd enough not to try to "fix".  But I'll see if I can use `super` here.

Reply:
`super` works - see https://github.com/linux-system-roles/network/pull/362/commits/e70dd1b04722fc2f617bdb2745f54b6312f56c18